### DISCUSSION TITLE
Unable to connect to Redshift Database after the new Version Update  Version 23.2.1.202309250303

### DISCUSSION THREAD

**Discussion by manikandan-macro** [dbeaver/dbeaver#21300] [DBeaver Development] [answered]
Unable to connect to Redshift Database after the new Version Update  Version 23.2.1.202309250303

We have used a based JDBC URL connection string to connect AWS Redshift DB, Please Check and correct it ASAP 

**Error** 

_sql error [42p01]: error: relation "pg_catalog.pg_enum" does not exist error: relation "pg_catalog.pg_enum" does not exist error: relation "pg_catalog.pg_enum" does not exist_

**Comment by LonwoLonwo** [comment, marked as answer]
Hello 

The issue is fixed here: https://github.com/dbeaver/dbeaver/issues/21293

Waiting for rerelease. For now, DBeaver version downgrading will help.
